File: CH3-desicion-sructure-and-boolean-logic/EX9-time-calculator/program-code/timeCalculator.js
const input = require("prompt-sync")();

let second;
let result;

second = input("Enter a second :");

if (second < 60) {
  result = second;
  console.log(result + " Second");
} else {
  if (second >= 60 && second < 3600) {
    result = second / 60;
    console.log(result + " Minute");
  } else {
    if (second >= 3600 && second < 86400) {
      result = second / 3600;
      console.log(result + " Hour");
    } else {
      result = second / 86400;
      console.log(result + " Day");
    }
  }
}

File: CH3-desicion-sructure-and-boolean-logic/EX9-time-calculator/timeCalculator.cpp
#include <iostream>
using namespace std;

int main()
{
    int second;
    int result;

    cout << "Enter a second :";
    cin >> second;

    if (second < 60)
    {
        result = second;
        cout << result << " Second" << endl;
    }
    else
    {
        if (second >= 60 && second < 3600)
        {
            result = second / 60;
            cout << result << " Minute" << endl;
        }
        else
        {
            if (second >= 3600 && second < 86400)
            {
                result = second / 3600;
                cout << result << " Hour" << endl;
            }
            else
            {
                result = second / 86400;
                cout << result << " Day" << endl;
            }
        }
    }
}